sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -40,14 +41,15 @@ class PrecisionRecallDisplay:
     figure_ : matplotlib Figure
         Figure containing the curve.
     """
-
-    def __init__(self, precision, recall, average_precision, estimator_name):
+    @_deprecate_positional_args
+    def __init__(self, *, precision, recall, average_precision, estimator_name):
```

Comment:
I think precision and recall should be positional. It's very likely to me that these objects will be passed as variable names and not need the contextualisation of a keyword name